src/main/java/com/iota/iri/utils/dag/TraversalCondition.java

```diff
@@ -0,0 +1,12 @@
+package com.iota.iri.utils.dag;
+
+import com.iota.iri.controllers.TransactionViewModel;
+
+/**
+ * Functional interface for the lambda function getting passed into the DAGUtils traverse method to check if the found
+ * transaction is relevant for our processing.
+ */
+@FunctionalInterface
+public interface TraversalCondition {
```

Comment:
I personally would have been fine if you would have used java's built in `Predicate` and named the variable traversalCondition (especially since you add such good javadocs).


However, I have no problem whatsoever with this code 👍 

Just telling you for the future if you want to save time :-)